bootcamp/tutorials/integration/llamaindex/llamaindex_milvus_full_text_search.ipynb

```diff
@@ -0,0 +1,437 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.    answer = query_engine.query("What did the author learn?")
</pre><p>this is a very bad example for full text search. no meaningful key word to match</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/milvus-io/bootcamp/pull/1516/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>